foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -69,10 +159,17 @@ def wait_for_job(job, job_class: type, start_time: datetime, loop_time: int = No
 # job in the database because it'd be stuck in a transaction.
 class NoOpEndToEndTestCase(TransactionTestCase):
     @tag("slow")
-    def test_no_op(self):
+    @vcr.use_cassette(
+        "/home/user/data_store/cassettes/surveyor.test_end_to_end.no_op.yaml",
```

Comment:
what about using a constant or something else to store the cassettes path? `/home/user/data_store/cassettes/`

Reply:
Oh yeah good call! When I originally did it like this I was planning on it but forgot.